docs/online.html

```diff
@@ -8,14 +8,18 @@
   "/vendor/codemirror/codemirror.css",
```

Comment:
Does this need to be vendored?  `https://unpkg.com/browse/lz-string@1.4.4/libs/lz-string.min.js` looks like it might work?

Reply:
We could load it directly from the CDN, but then it wouldn't work in the tests anymore, because we reference the library directly there. I don't believe there is any way in Jest/Node to load from URLs directly, unfortunately. Another advantage of vendoring is that I changed the library to use standard `import`/`export` syntax rather than CommonJS.

(In the long run, we should just `npm install` this and bundle it, but I didn't want to do that in this PR.)